x/evm/keeper/msg_server.go

```diff
@@ -801,6 +801,7 @@ func (s msgServer) VoteConfirmDeposit(c context.Context, req *types.VoteConfirmD
 	}
 	event = event.AppendAttributes(sdk.NewAttribute(sdk.AttributeKeyAction, types.AttributeValueConfirm))
 
+	// check casting
```

Comment:
What does this comment mean?

Reply:
Oops, that was for me. The subsequent statement panics if we don't restrict the original confirm deposit amount to be below `Int64`.